gateway/1.4/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc

```diff
@@ -5,27 +5,35 @@ endif::[]
 :imagesdir: ../assets/images
 :page-mode: conn
 
-// intro and graphics for each option - linux, docker, k8s
-include::partial$task-config-logs-third-party.adoc[tags=intro1;intro2;icon-table;byb;file-example]
-
-// configure log output for Flex Gateway as a Linux Service
-include::partial$task-config-logs-third-party.adoc[tags=linux;sample-config-all-intro]
-+
-include::partial$task-config-logs-third-party.adoc[tags=sample-config-all;view-logs-conn]
-
-// configure log output for Flex Gateway in a Docker container
-include::partial$task-config-logs-third-party.adoc[tags=docker-intro]
-+
-include::partial$task-reg-run-flex-gateway.adoc[tags=start-command-local]
-+
-include::partial$task-config-logs-third-party.adoc[tags=sample-config-all-intro]
-+
-include::partial$task-config-logs-third-party.adoc[tags=sample-config-all;view-logs-conn]
-
-//configure log output for Flex Gateway as an K8s Ingress Controller
-include::partial$task-config-logs-third-party.adoc[tags=k8s;sample-config-all-intro]
-
-include::partial$task-config-logs-third-party.adoc[tags=sample-config-all;view-logs-conn]
+include::partial$task-config-logs-third-party.adoc[tags=intro1;intro2-conn]
+
+include::partial$task-config-logs-third-party.adoc[tags=icon-table]
+
+include::partial$task-config-logs-third-party.adoc[tags=byb]
+
+include::partial$task-config-logs-third-party.adoc[tags=runtime-access-logs]
+
+include::partial$task-config-logs-third-party.adoc[tags=file-example]
+
+include::partial$task-config-logs-third-party.adoc[tags=linux;view-logs-conn;configure-message-logging-step]
+
+include::partial$task-config-logs-third-party.adoc[tags=docker;view-logs-conn;configure-message-logging-step]
+
+include::partial$task-config-logs-third-party.adoc[tags=k8s;view-logs-conn;configure-message-logging-step]
+
+include::partial$task-config-logs-third-party.adoc[tags=file-configuration-example]
+
+include::partial$task-config-logs-third-party.adoc[tags=http-configuration-example]
+
+include::partial$task-config-logs-third-party.adoc[tags=splunk-configuration-example]
+
+[[configure-access-logs]]
+== Configure Access Logs
+To configure access logs in your log output, you must apply a Message Logging policy to the APIs whose request and response data you want to monitor. If you have previously applied a Message Logging policy to an API instance, you do not need to configure another. 
```

Comment:
I stumbled over "whose", which is technically OK per the SR.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22whose%22whenreferringtoinanimateobjects